Validation/Configuration/python/globalValidation_cff.py

```diff
@@ -120,11 +120,15 @@
 )
 globalValidationTrackingOnly = cms.Sequence()
 
-def _modifyGlobalValidationForPhase2( theProcess ):
-    theProcess.load('Validation.Configuration.gemSimValid_cff')
-    theProcess.load('Validation.Configuration.me0SimValid_cff')
-    theProcess.globalValidation += theProcess.gemSimValid
-    theProcess.globalValidation += theProcess.me0SimValid
+from Validation.Configuration.gemSimValid_cff import *
+from Validation.Configuration.me0SimValid_cff import *
+
+_run3_globalValidation = globalValidation.copy()
+_run3_globalValidation += gemSimValid
+
+_phase2_globalValidation = _run3_globalValidation.copy()
+_phase2_globalValidation += me0SimValid
 
 from Configuration.StandardSequences.Eras import eras
-modifyConfigurationStandardSequencesGlobalValidationForPhase2_ = eras.phase2_muon.makeProcessModifier( _modifyGlobalValidationForPhase2 )
+eras.run3_GEM.toReplaceWith( globalValidation, _run3_globalValidation )
+eras.phase2_muon.toReplaceWith( globalValidation, _phase2_globalValidation )
```

Comment:
@Dr15Jones is this the most practical way of dealing with modifications of the same thing by a "sequence" of  modifiers?
Here we have ``` Phase2 = (phase2_muon, run3_GEM) ```
and both of them modify the same globalValidation by replacing it with something else. Is it guaranteed that phase2_muon is executed first and takes out the original globalValidation object so that the run3_GEM doesn't do anything with it?